### PR TITLE
Update what-are-truthy-and-falsy-values.md

### PR DESCRIPTION
**Pull request by Yash-Handa**
Hey @chowdhurian and @fhemberger,
In accordance with issue #1977, I have updated the Article **what-are-truthy-and-falsy-values** with the following changes:

* Added better syntax high lighting.
* some other miscellaneous updates and refactors.

If there are any **suggestion/enhancement/tips** please feel free to tell me, I would love to incorporate those changes in this Article 😄

NOTE: The article was completely relevant and didn't require any major updates.

### CODE REVIEW

File: locale/en/knowledge/javascript-conventions/what-are-truthy-and-falsy-values.md [hunk 1]
```diff
@@ -15,60 +15,68 @@ JavaScript is weakly typed language. That means different types can be
...
-    console.log("1" > 0); // true, "1" converted to number
-    console.log(1 + "1"); // 11, 1 converted to string
+```js
+console.log("1" > 0); // true, "1" converted to number
+console.log(1 + "1"); // 11, 1 converted to string
+```
 
-Type conversion also applys when values are used in unary boolean
+Type conversion also applies when values are used in unary boolean
 operations, most notably if statements. If a value converts to the
```
Comment (Maledong) [inline locale/en/knowledge/javascript-conventions/what-are-truthy-and-falsy-values.md]: "converts" => "is converted"

File: locale/en/knowledge/javascript-conventions/what-are-truthy-and-falsy-values.md [hunk 2]
```diff
@@ -15,60 +15,68 @@ JavaScript is weakly typed language. That means different types can be
...
 boolean true, then it is said to be "truthy". If it converts to false
```
Comment (Maledong) [inline locale/en/knowledge/javascript-conventions/what-are-truthy-and-falsy-values.md]: "converts" => "is converted"

File: locale/en/knowledge/javascript-conventions/what-are-truthy-and-falsy-values.md (hunk 1 above)
Comment (ghost) [inline locale/en/knowledge/javascript-conventions/what-are-truthy-and-falsy-values.md:24]: "converts"=>"is converted"

Comment (ghost) [inline locale/en/knowledge/javascript-conventions/what-are-truthy-and-falsy-values.md:24]: "converts"=>"is converted"

**Comment by ghost** [comment]
Others look good, thanks!

**Review by willin — APPROVED** [review]

File: locale/en/knowledge/javascript-conventions/what-are-truthy-and-falsy-values.md (hunk 1 above)
Comment (fhemberger) [reply locale/en/knowledge/javascript-conventions/what-are-truthy-and-falsy-values.md:24]: "converts" is correct here.

Comment (fhemberger) [reply locale/en/knowledge/javascript-conventions/what-are-truthy-and-falsy-values.md:24]: "converts" is correct here.

**Review by fhemberger — APPROVED** [review]

**Comment by fhemberger** [comment]
Thanks!

**Pull request merged**